classes/issue/Issue.inc.php

```diff
@@ -559,6 +559,14 @@ function hasDescription() {
 	function getDAO() {
 		return DAORegistry::getDAO('IssueDAO');
 	}
+
+	/**
+	 * Display the object in Import/Export results
+	 * @return string A string that Identifies the object
+	 */
+	function getUIDisplayString() {
+		return $this->getId() . ' - ' . $this->getIssueIdentification();
```

Comment:
(As in the pkp-lib PR: this should probably be a locale key.)

Reply:
This one still needs attention.

Reply:
Fixed